python/ray/autoscaler/v2/instance_manager/instance_manager.py

```diff
@@ -315,3 +321,130 @@ def get_status_times_ns(
             ts_list.append(status_update.timestamp_ns)
 
         return ts_list
+
+
+class DefaultInstanceManager(InstanceManager):
```

Comment:
would we ever create another instance manager? if not we can just remove the interface